src/transformers/models/reformer/modeling_reformer.py

```diff
@@ -2194,12 +2195,7 @@ def set_output_embeddings(self, new_embeddings):
         self.lm_head.decoder = new_embeddings
 
     @add_start_docstrings_to_model_forward(REFORMER_INPUTS_DOCSTRING)
-    @add_code_sample_docstrings(
-        processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
-        output_type=CausalLMOutput,
-        config_class=_CONFIG_FOR_DOC,
-    )
+    @replace_return_docstrings(output_type=CausalLMOutput, config_class=_CONFIG_FOR_DOC)
```

Comment:
The reason for using `replace_return_docstrings` here is below:
- We need to load the model with is_decoder=True.  (line:2230)